packages/nextjs/src/config/webpack.ts

```diff
@@ -163,6 +164,29 @@ export function constructWebpackConfigFunction(
         ],
       });
 
+      let vercelCronsConfig: VercelCronsConfig = undefined;
+      try {
+        if (process.env.VERCEL && userSentryOptions.automaticVercelMonitors !== false) {
+          // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+          vercelCronsConfig = JSON.parse(fs.readFileSync(path.join(process.cwd(), 'vercel.json'), 'utf8')).crons;
+          logger.info(
+            `${chalk.cyan(
+              'info',
+            )} - Creating Sentry cron monitors for your Vercel Cron Jobs. You can disable this feature by setting the ${chalk.bold.cyan(
```

Comment:
m: Should we only log this out if the `crons` field is defined? Otherwise I can see how this might get confusing

Reply:
Good point!